### PR TITLE
Add rancher max version of 2.5.99-0

### PR DESCRIPTION
**Pull request by thedadams**
This is the list of charts that were changed:
longhorn
rancher-istio
rancher-alerting-drivers
rancher-monitoring
rancher-webhook
rancher-cis-benchmark
rancher-vsphere-cpi
rancher-vsphere-csi

### CODE REVIEW

File: charts/longhorn-1.1/longhorn-crd/1.1.200+up1.1.2/Chart.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 annotations:
+  catalog.cattle.io/rancher-version: <2.5.99-0
```
Comment (jiaqiluo) [inline charts/longhorn-1.1/longhorn-crd/1.1.200+up1.1.2/Chart.yaml:2]: forgot to mention that we do not need to add the annotation to the CRD charts and non-user-facing chart.

@cbron @aiyengar2 Please correct me if I am wrong.

Comment (aiyengar2) [reply charts/longhorn-1.1/longhorn-crd/1.1.200+up1.1.2/Chart.yaml:2]: Yes we don't need to add it to those.

**Review by aiyengar2 — CHANGES_REQUESTED** [review]
Based on list in https://github.com/rancher/rancher/issues/33528.

- longhorn 1.1.200+up1.1.2
  - [x] chart updated?
  - [x] asset updated?
  - [x] index.yaml updated?
- longhorn 1.2.200+up1.2.2
  - [x] chart updated?
  - [ ] asset updated?
  - [ ] index.yaml updated?
- rancher-alerting-drivers 1.0.200
  - [x] chart updated?
  - [x] asset updated?
  - [x] index.yaml updated?
- rancher-backup 1.2.0
  - [ ] chart updated?
  - [ ] asset updated?
  - [ ] index.yaml updated?
- rancher-cis-benchmark 1.0.600
  - [x] chart updated?
  - [x] asset updated?
  - [x] index.yaml updated?
- rancher-istio 1.9.800
  - [x] chart updated?
  - [x] asset updated?
  - [x] index.yaml updated?
- rancher-monitoring 16.6.0
  - [x] chart updated?
  - [x] asset updated?
  - [x] index.yaml updated?
- rancher-vsphere-cpi 1.0.100
  - [x] chart updated?
  - [x] asset updated?
  - [x] index.yaml updated?
- rancher-vsphere-csi 2.3.0
  - [x] chart updated?
  - [x] asset updated?
  - [x] index.yaml updated?
- rancher-webhook 0.1.200+up0.1.2
  - [x] chart updated?
  - [x] asset updated?
  - [x] index.yaml updated?

Changes to longhorn `1.2.200+up1.2.2` and `rancher-backup 1.2.0` need to be made.

longhorn-crd index.yaml change needs to be removed.

rancher-monitoring-crd timestamp change should be removed.

rancher-cis-benchmark-crd asset should not be modified.

Once these fixes are made, at least one of us reviewers also need to do a deep check on the tgz files pushed here to ensure that the assets have been updated as expected. I can do that in the final review before merge, once all other reviewers have approved.

File: charts/longhorn-1.1/longhorn/1.1.200+up1.1.2/Chart.yaml [hunk 2]
```diff
@@ -3,6 +3,7 @@ annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: Longhorn
   catalog.cattle.io/namespace: longhorn-system
+  catalog.cattle.io/rancher-version: <2.5.99-0
```
Comment (cbron) [inline charts/longhorn-1.1/longhorn/1.1.200+up1.1.2/Chart.yaml:6]: This is a weird one as longhorn supports both 1.1.x and 1.2.x in 2.6, but I think its fine as `longhorn-100.0.0+up1.1.2.tgz` exists in 2.6. We'd want them to move to the 100's and not freshly install this. Same for chart below.

File: charts/rancher-alerting-drivers/rancher-alerting-drivers/1.0.200/Chart.yaml [hunk 3]
```diff
@@ -1,4 +1,5 @@
 annotations:
+  catalog.cattle.io/rancher-version: <2.5.99-0
```
Comment (cbron) [inline charts/rancher-alerting-drivers/rancher-alerting-drivers/1.0.200/Chart.yaml:2]: Yes, 100.0.0 exists

File: charts/rancher-cis-benchmark/rancher-cis-benchmark/1.0.600/Chart.yaml [hunk 4]
```diff
@@ -1,4 +1,5 @@
 annotations:
+  catalog.cattle.io/rancher-version: <2.5.99-0
```
Comment (cbron) [inline charts/rancher-cis-benchmark/rancher-cis-benchmark/1.0.600/Chart.yaml:2]: yes, 2.0 exists

File: charts/rancher-istio-1.9/rancher-istio/1.9.800/Chart.yaml [hunk 5]
```diff
@@ -4,6 +4,7 @@ annotations:
   catalog.cattle.io/display-name: Istio
   catalog.cattle.io/namespace: istio-system
   catalog.cattle.io/os: linux
+  catalog.cattle.io/rancher-version: <2.5.99-0
```
Comment (cbron) [inline charts/rancher-istio-1.9/rancher-istio/1.9.800/Chart.yaml:7]: yes, 1.10 exists

File: charts/rancher-monitoring/rancher-monitoring/16.6.0/Chart.yaml [hunk 6]
```diff
@@ -5,6 +5,7 @@ annotations:
     - name: Upstream Project
       url: https://github.com/prometheus-operator/kube-prometheus
   artifacthub.io/operator: "true"
+  catalog.cattle.io/rancher-version: <2.5.99-0
```
Comment (cbron) [inline charts/rancher-monitoring/rancher-monitoring/16.6.0/Chart.yaml:8]: yes 100 exists

File: charts/rancher-vsphere-cpi/rancher-vsphere-cpi/1.0.100/Chart.yaml [hunk 7]
```diff
@@ -1,4 +1,5 @@
 annotations:
+  catalog.cattle.io/rancher-version: <2.5.99-0
```
Comment (cbron) [inline charts/rancher-vsphere-cpi/rancher-vsphere-cpi/1.0.100/Chart.yaml:2]: yes, 100 exists

File: charts/rancher-vsphere-csi/rancher-vsphere-csi/2.3.0/Chart.yaml [hunk 8]
```diff
@@ -1,4 +1,5 @@
 annotations:
+  catalog.cattle.io/rancher-version: <2.5.99-0
```
Comment (cbron) [inline charts/rancher-vsphere-csi/rancher-vsphere-csi/2.3.0/Chart.yaml:2]: This one I actually think might need to stay ? @PennyScissors when we OOB these they will be 100.x.x right ? If so then this is good.

File: charts/rancher-webhook/rancher-webhook/0.1.200+up0.1.2/Chart.yaml [hunk 9]
```diff
@@ -1,4 +1,5 @@
 annotations:
+  catalog.cattle.io/rancher-version: <2.5.99-0
```
Comment (cbron) [inline charts/rancher-webhook/rancher-webhook/0.1.200+up0.1.2/Chart.yaml:2]: yes, we have 1.x

**Review by cbron — COMMENTED** [review]
lgtm except for one question

**Review by aiyengar2 — APPROVED** [review]
LGTM

**Pull request merged**